### PR TITLE
[12.0][MIG] purchase_stock_picking_restrict_cancel

### PR DESCRIPTION
**Pull request by kittiu**
Migration to v12 (WIP)

Dependencies
- [x] stock_picking_restrict_cancel_with_orig_move https://github.com/OCA/stock-logistics-workflow/pull/515

### CODE REVIEW

**Comment by kittiu** [comment]
This only need to rebase after its dependencies

**Comment by kittiu** [comment]
Rebased.
Now ready for review.

**Review by rousseldenis — APPROVED** [review]
Code review

File: purchase_stock_picking_restrict_cancel/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# Copyright 2018 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+{
+    "name": "Purchase Stock Picking Restrict Cancel",
+    "summary": "Restrict cancellation with Purchase.",
+    "version": "12.0.1.0.1",
```
Comment (tbaden) [inline purchase_stock_picking_restrict_cancel/__manifest__.py]: please use the version: 12.0.1.0.0

**Review by tbaden — CHANGES_REQUESTED** [review]
thanks for the work 👍 
I did a code review: LGTM
just a minor change

File: purchase_stock_picking_restrict_cancel/__manifest__.py (hunk 1 above)
Comment (kittiu) [reply purchase_stock_picking_restrict_cancel/__manifest__.py]: Fixed. Thank you!

**Review by tbaden — APPROVED** [review]

**Pull request merged**